### PR TITLE
[DR-2174] Removed hardcoded filters from the databrowser page

### PR DESCRIPTION
**Pull request by lu-c**
Pulling them out from the catalog data instead

<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

**Comment by kyuksel** [comment]
@lu-c FYI that Terra UI PRs need ticket prefix (e.g. `[DR-2116]`) for compliance purposes.

[DR-2116]: https://broadworkbench.atlassian.net/browse/DR-2116?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by kyuksel** [comment]
@lu-c We are short-staffed today due to OOOs, and may not be able to get to your PR. Sorry for any delays.

**Review by dmohs — APPROVED** [review]

File: src/pages/library/DataBrowser.js [hunk 1]
```diff
@@ -34,90 +34,79 @@ const styles = {
...
+      ],
+      Pending: [
+        div({ style: { display: 'flex' } }, [
+          icon('lock', { style: { color: styles.access.pending, marginRight: 5 } }),
+          div(['Pending'])
+        ])
+      ]
+    }
+  }, {
+    name: 'Consortium',
```
Comment (petesantos) [inline src/pages/library/DataBrowser.js:73]: How would you feel about using a helper function for these - something like:
```javascript
const getUnique = (prop, data) => _.flow(_.flatMap(prop), _.compact, _.uniq, _.sortBy(_.toLower))(data)
```

File: src/pages/library/DataBrowser.js [hunk 2]
```diff
@@ -34,90 +34,79 @@ const styles = {
...
+  }, {
+    name: 'Data Type',
+    labels: _.flow(
+      _.flatMap('dataType'),
+      _.uniqBy(_.identity),
+      _.sortBy(_.toLower)
+    )(dataCatalog)
+  }, {
+    name: 'File Type',
+    labels: _.flow(
```
Comment (petesantos) [inline src/pages/library/DataBrowser.js]: For this one you could call the helper as:
```getUnique('dcat:mediaType', _.flatMap('files', dataCatalog))```

File: src/pages/library/DataBrowser.js [hunk 3]
```diff
@@ -34,90 +34,79 @@ const styles = {
...
+      _.flatMap(c => c.files),
+      _.map('dcat:mediaType'),
+      _.uniqBy(_.identity),
+      _.sortBy(_.lowerCase)
+    )(dataCatalog)
+  }, {
+    name: 'Disease',
+    labels: _.flow(
```
Comment (petesantos) [inline src/pages/library/DataBrowser.js]: for these it would look like:
```getUnique('samples.disease', dataCatalog)```

**Comment by kyuksel** [comment]
@lu-c I kicked off the failing integration test one last time. Looking at the PR changes and the test failure signature, I think this PR doesn't need to be held up due to the test failure (if it persists).

File: src/pages/library/DataBrowser.js (hunk 1 above)
Comment (lu-c) [reply src/pages/library/DataBrowser.js:73]: sure, i like it, let me make the change 👍

**Review by petesantos — APPROVED** [review]

**Comment by kyuksel** [comment]
@lu-c FYI that this PR is now good to merge 😊

**Pull request merged**